wallet/how-to/manage-networks/add-network.md

```diff
@@ -30,6 +30,12 @@ The confirmations look like the following:
     </div>
 </div>
 
+:::note
+
+You can also [add a Hardhat development network to MetaMask](../how-to/run-devnet.md)
```

Comment:
```suggestion
You can also [add a development network](../how-to/run-devnet.md), such as Hardhat, to MetaMask.
```

To address Christian's comment on generalizing the note.

Reply:
Hi @alexandratran I proposed a slightly different wording.